Source/FakeItEasy/Core/IFakeObjectOptionsBuilder.cs

```diff
@@ -0,0 +1,18 @@
+namespace FakeItEasy.Core
+{
+    using System;
+    using Creation;
+
+    /// <summary>
+    /// Handles global configuration of fake objects by building fake creation options.
+    /// </summary>
+    public interface IFakeObjectOptionsBuilder

```

Comment:
Probably off-topic for this PR, but I'm curious... what's the difference between a `IFakeObjectOptionsBuilder` and a `IFakeOptionsBuilder`?